src/nvim/popupmenu.c

```diff
@@ -1321,6 +1324,14 @@ static void pum_position_at_mouse(int min_width)
   int col = mouse_col;
   pum_win_row_offset = 0;
   pum_win_col_offset = 0;
+
+  if (ui_has(kUIMultigrid) && (grid == 0 || grid == 1)) {
+    win_T *wp = mouse_find_win(&grid, &row, &col);
+    // mouse_find_win compensates for the offset due to winbar for example
+    // but we don't want that here
```

Comment:
punctuation is a cheap way to avoid ambiguity in comments. line-length max is 100.
```suggestion
    // mouse_find_win compensates for 'winbar' offset (for example), but we don't want that here.
```

Reply:
fixed